plugins/woocommerce/src/StoreApi/Routes/V1/ProductCollectionData.php

```diff
@@ -132,31 +169,31 @@ public function get_collection_params() {
 		$params = ( new Products( $this->schema_controller, $this->schema ) )->get_collection_params();
 
 		$params['calculate_price_range'] = [
-			'description' => __( 'If true, calculates the minimum and maximum product prices for the collection.', 'woocommerce' ),
+			'description' => __( 'If true, calculates the minimum and maximum product prices for the collection.', 'woo-gutenberg-products-block' ),
```

Comment:
I believe the text domain should be `woocommerce` right (since the merge)?

Reply:
Nice catch! I just fixed this in https://github.com/woocommerce/woocommerce/pull/45247/commits/6e473aded759a4e5949328c25d6d5f75913efa39